docs/compatibility-chart.en.md

```diff
@@ -29,10 +29,11 @@ Carrier not being listed on the table does not mean that this patch won't work f
 | South America | Peru (716)              | Claro (10)          | O                   | X                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/discussions/93)                                                                                                                      |
 | Europe        | Bulgaria (284)          | Yettel (05)         | O                   | Not operational                         | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/discussions/1#discussioncomment-5012767)                                                                                             |
 | Europe        | Czech Republic (230)    | O2 (02)             | O                   | O                                       | Tested on my own Pixel 6a, both VoLTE & VoWiFi work [Reference](https://github.com/kyujin-cho/pixel-volte-patch/pull/130) |
+| Europe        | France (208)   | Bouygues (20)             | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/175) |
 | Europe        | Various countries       | Elisa               | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/125#issue-1698533656)                                                                                                         |
 | Europe        | Various countries       | Saunalahti          | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/125#issue-1698533656)                                                                                                         |
 | Europe        | Switzerland (228)       | Salt (03)           | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/125#issuecomment-1546696568)                                                                                                  |
-| Europe        | Poland (260)            | Orange PL (03)      | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/17)                                                                                                                           |
+| Europe        | Poland (260)            | Orange PL (03)      | O                   | O                                       | [Reference](https://github.com/kyujin-cho/pixel-volte-patch/issues/17). If Vo-Wifi isn't working, you may need to add the `ims` APN [Reference](https://github.com/kyujin-cho/pixel-volte-patch/discussions/1#discussioncomment-6261820)                                                                                                                          |
```

Comment:
wait should I change the link to
https://github.com/kyujin-cho/pixel-volte-patch/issues/136#issuecomment-1565598716

Reply:
probably because that was turkcell.